### ISSUE TITLE
Change initializations to C++11 style

### ISSUE THREAD

**Issue by stiber** [UWB-Biocomputing/Graphitti#155]
I.e.,  using "{}"

**Comment by torisalvatore** [comment]
requires manually going through every init of an array in the simulator. good first project.

**Comment by stiber** [comment]
And that connects to #24, switching from arrays to vectors.

**Comment by VivekGandhi54** [comment]
I don't think you can use initializer lists at all for any heap arrays.
`std::fill_n(array, count, value)` sets the first 'count' elements of the array to 'value', which is a better option for heap arrays.

**Comment by stiber** [comment]
Address this after #24, since many arrays will change to vectors.

**Comment by jardiamj** [comment]
Hi @KamathScience! Would you happen to know if this issue is still relevant? I believe that the only arrays that are left, on the CPU side, are the `boolean` ones.

**Comment by KamathScience** [comment]
Yep, we only have three boolean arrays left on the CPU side (`inUse` from `AllEdges`, `isAvailable` from `All911Edges`, and `masks` from `SInputPossion`).

**Comment by stiber** [comment]
There might be other variables initialized using "old" `()` and `=` syntax.
